draft-ietf-masque-connect-ip.md

```diff
@@ -113,6 +113,7 @@ works to exclude it in the syntax.
 Examples are shown below:
 
 ~~~
+https://example.org/masque/{target}/{ipproto}/
```

Comment:
Can you also add the default template to the example?

Reply:
Oh whoops, I should have done it with this one

Reply:
```suggestion
https://example.org/.well-known/masque/ip/{target}/{ipproto}/
```